doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -127,15 +149,10 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.asr.gen.01` | General | The Architecture shall/must xxx. |
-| `req.asr.gen.02` | General | The Architecture shall/must xxx. |
-| `req.asr.gen.03` | General | The Architecture shall/must xxx. |
-| `req.asr.gen.01` | General | The Architecture shall/must xxx. |
-| `req.asr.gen.02` | General | The Architecture shall/must xxx. |
-| `req.asr.gen.03` | General | The Architecture shall/must xxx. |
 | `req.asr.ing.01` | Integration | The Architecture **must** support integration with the standard OSS/BSS assurance systems and will support automation of assurance processes. |
```

Comment:
"the standard OSS/BSS assurance systems" is undefined and includes the risk to bring legacy system requirements/dependencies into the cloud environment. Aren't the requirements req.gen.ost.01, req.inf.ntw.12, and especially req.asr.mon.01 already sufficient?

Reply:
I agree, that probably this one is redundant now.

Reply:
OK, removed now.